lib/framework/stub/contacts_manager.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:ensemble/framework/error_handling.dart';
+
+typedef ContactSuccessCallback = void Function(
+    List<Map<String, dynamic>> contacts);
```

Comment:
Can we have stronger typing on contacts?

Reply:
@evshi I need to have an exact clone of https://pub.dev/documentation/flutter_contacts/latest/contact/Contact-class.html then, we no need to use the Contact class (because it's from flutter_contacts). So can I do an exact clone of it with all nested classes with properties or just create a class which exposes only phone numbers?